samples/servicecontrol/adapter-sqlserver/sample.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Using adapter to centralize monitoring of multi-instance SQL Server
+component: SCTransportAdapter
+reviewed: 2017-05-11
+related:
+ - servicecontrol
+ - servicecontrol/plugins
+---
+
+
+The purpose of the adapter is to isolate ServiceControl from specifics of physical deployment topology of the business endpoints (such as [multi-instance](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) mode. In order to do so, the adapter provides a ServiceControl interface to the business endpoints.
+
```

Comment:
I'd also add "This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using SQL Server transport in a multi-database mode."